File: m2cgen/interpreters/rust/linear_algebra.rs
fn add_vectors(v1: Vec<f64>, v2: Vec<f64>) -> Vec<f64> {
    v1.iter().zip(v2.iter()).map(|(&x, &y)| x + y).collect::<Vec<f64>>()
}
fn mul_vector_number(v1: Vec<f64>, num: f64) -> Vec<f64> {
    v1.iter().map(|&i| i * num).collect::<Vec<f64>>()
}

File: m2cgen/interpreters/elixir/linear_algebra.ex
defp add_vectors(v1_list, v2_list) do
  for {a,b} <- Enum.zip(v1_list, v2_list), do: a+b
end

defp mul_vector_number(v1, num) do
  for i <- v1, do: i * num
end
